### PR TITLE
Enable numberOfIntervals run option

### PR DESCRIPTION
**Pull request by nllong**
#### Any background context you want to provide?
We were only using `stepSize` argument when running openmodelica,

#### What does this PR accomplish?
* Add `number_of_intervals` as a simulation option. The simulation will favor `stepSize` if both are given.

#### How should this be manually tested?
* unit tests should suffice

#### What are the relevant tickets?
n/a

### CODE REVIEW

File: geojson_modelica_translator/modelica/modelica_runner.py [hunk 1]
```diff
@@ -102,19 +102,26 @@ def _copy_over_docker_resources(self, run_path: Path, filename: Union[str, Path,
         the generation of the OpenModelica scripts to load and compile/run
         the simulation.
 
-        If passing the start, stop, and step times, then at least start and stop must be supplied to
-        update the simuation to use these values instead of the defaults.
+        Must pass start_time, stop_time, and either step_size or number_of_intervals.
```
Comment (vtnate) [inline geojson_modelica_translator/modelica/modelica_runner.py:105]: We're requiring all these now?

File: geojson_modelica_translator/modelica/modelica_runner.py [hunk 2]
```diff
@@ -102,19 +102,26 @@ def _copy_over_docker_resources(self, run_path: Path, filename: Union[str, Path,
...
 
         Args:
             run_path (Path): Path where the model will be run, this is where the files will be copied.
             filename (str): name of the file that will be loaded (e.g., BouncingBall.mo, package.mo)
             model_name (str): name of the model to run (e.g., BouncingBall, Districts.DistrictModel)
+            **kwargs: Arbitrary keyword arguments.
```
Comment (vtnate) [inline geojson_modelica_translator/modelica/modelica_runner.py:111]: Why use kwargs instead of explicitly setting them as parameters of this method?

**Review by vtnate — APPROVED** [review]
Jinja for life!! 🤣 Couple questions about kwargs for my own education. This looks good.

File: geojson_modelica_translator/modelica/modelica_runner.py (hunk 2 above)
Comment (nllong) [reply geojson_modelica_translator/modelica/modelica_runner.py:111]: i waffled on this too. I do allow none of these args to be passed and OMC will just run with the defaults.

File: geojson_modelica_translator/modelica/modelica_runner.py (hunk 1 above)
Comment (nllong) [reply geojson_modelica_translator/modelica/modelica_runner.py:105]: not requiring.. we can pass none

**Pull request merged**